pkg/service/test/util.go

```diff
@@ -168,7 +167,6 @@ func Flatten(j interface{}) map[string][]string {
 			}
 		}
 	default:
-		fmt.Println(Emoji, "found invalid value in json", j, x.Kind())
```

Comment:
We have to provide some error statement here right and error handling if the kind is not handled in the switch case

Reply:
again as the above file, will have to pass the logger for 3 layers